### DISCUSSION TITLE
Postcodes

### DISCUSSION THREAD

**Discussion by MrMikeRose** [digital-land/data-standards-backlog#71] [Planning consideration]
A discussion about the [postcodes](https://design.planning.data.gov.uk/planning-consideration/postcodes) planning consideration.

**Comment by MrMikeRose** [comment]
I have begun this one - it is a little more complicated than a single glance as there are 2 datasets, one open and one not, and further research depends upon which would be needed for planning (if either!) and WHAT it would be needed for.

**Reply by psd** [reply]
The main source to review from the IP perspective is the [Postcode Directory (ONSPD)](https://www.ons.gov.uk/methodology/geography/geographicalproducts/postcodeproducts)

**Comment by psd** [comment]
Needs for postcode data:

1. datasets containing address data can contain postcode fields. Having a list of valid postcodes links and groups this data
2. postcodes are a convenient way for users to find data, so searching for entities close to where a postcode applies and centering the map at the centre-point for a postcode are two needs for users of the platform

**Comment by psd** [comment]
Note, the name used in ONS, OS, Royal Mail products and documentation is `postcode`. 

Synonyms include [postal code](https://en.wikipedia.org/wiki/Postal_code) and `post code`. 

Wikipedia's [Postcodes in the United Kingdom](https://en.wikipedia.org/wiki/Postcodes_in_the_United_Kingdom) says:

> [Postal codes](https://en.wikipedia.org/wiki/Postal_code) used in the United Kingdom, [British Overseas Territories](https://en.wikipedia.org/wiki/British_Overseas_Territories) and [Crown dependencies](https://en.wikipedia.org/wiki/Crown_dependencies) are known as postcodes (originally, postal codes).

**Comment by psd** [comment]
I've created a postcode [field](https://digital-land.github.io/specification/field/postcode/) and [dataset](https://digital-land.github.io/specification/dataset/postcode/) in the specification
